client/datatypes.go

```diff
@@ -2,9 +2,9 @@
 // User Types
 //
 // Generated with goagen v0.2.dev, command line:
-// $ goagen
+// $ goagen.exe
```

Comment:
That is ok, except if the workspace does not compile after the regen, 
I'll have to fix the errors first.


Am 24.06.2016 um 13:36 schrieb Aslak Knutsen:
>
> In client/datatypes.go 
> <https://github.com/almighty/almighty-core/pull/4#discussion_r68386328>:
>
> > @@ -2,9 +2,9 @@
> >  // User Types
> >  //
> >  // Generated with goagen v0.2.dev, command line:
> > -// $ goagen
> > +// $ goagen.exe
>
>  1. Design change
>  2. Regenerated commit
>  3. Impl commit
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/almighty/almighty-core/pull/4/files/6027d22ed35050dfc95fdf2ea1abf15d3763e94a..90e1426b9cf7e73b8bbed3b7aa447293d855f203#r68386328>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/AMjc-uFD9DM7a-4iJS0fvl0cPCOxpXHtks5qO8FRgaJpZM4I8524>.
>